x-pack/platform/packages/shared/file-upload-common/src/types.ts

```diff
@@ -19,4 +19,13 @@ export interface OpenFileUploadLiteContext {
   indexSettings?: IndicesIndexSettings;
   autoAddInference?: string;
   autoCreateDataView?: boolean;
+  initialIndexName?: string;
+  flyoutContent?: FlyoutContent;
+}
+
+export interface FlyoutContent {
+  title?: string;
+  description?: string;
```

Comment:
Both `title` and `description` should support React elements.
	•	`FormattedMessage` is the preferred way to handle i18n in Kibana.
	•	Consumers may want to pass a custom component that includes links or other elements.

```suggestion
  title?: string | ReactNode;
  description?: string | ReactNode;
```

Reply:
Updated in https://github.com/elastic/kibana/pull/213275/commits/c19ef736fe8052616736eedb6757c01c4c5df97b